grpc/oob/bmc/bmc.go

```diff
@@ -113,6 +113,14 @@ func NewBMCResetter(opts ...Option) (*Action, error) {
 	return a, nil
 }
 
+func (m Action) closeConnection(ctx context.Context, connections []oob.BMC) {
```

Comment:
why not `closeConnections` since there's seemingly nothing stopping it having more than one connection open?